src/js/controller/controller.js

```diff
@@ -212,6 +212,10 @@ Object.assign(Controller.prototype, {
                 }
             });
 
+            if (_model.get('mute') && _model.get('enableCaptionsOnMute')) {
+                _captions.enableOnLoad = true;
```

Comment:
Can you just `model.set('captionsIndex', 0);` here, instead of having this flag?